relationships/candidates/AWSMQEXCHANGE.yml

```diff
@@ -0,0 +1,18 @@
+category: AWSMQEXCHANGE
+lookups:
+  - entityTypes:
+      - domain: INFRA
+        type: AWSMQEXCHANGE
+    tags:
+      matchingMode: ALL
+      predicates:
+        - tagKeys: ["aws.mq.configurationEndpointAddress"]
+          field: endpoint
+        - tagKeys: ["aws.mq.configurationEndpointPort"]
+          field: port
+    onMatch:
+      onMultipleMatches: RELATE_ALL
+    onMiss:
+      action: CREATE_UNINSTRUMENTED
+      uninstrumented:
```

Comment:
I think we don't have that uninstrumented type yet and would need to add a definition for it.